database/helper.js

```diff
@@ -0,0 +1,20 @@
+let percentCalc = (reviews) => {
+
+  let recommended = 0;
+  let percent;
```

Comment:
I've heard/read that this is not a super great practice (to create a variable and not assign it a type, i.e. Number, String, etc. you may want to say something like let percent = new Number(); or let percent =  0 to avoid varible hijacking (this is not the official term, but it's what I've quated to the situation)